tutoraspects/plugin.py

```diff
@@ -46,9 +46,12 @@
         ),
         (
             "EVENT_SINK_CLICKHOUSE_MODELS",
-            ["course_overviews", "user_profile", "external_id"],
+            ["course_overviews", "user_profile", "external_id", "auth_user"],
+        ),
+        (
+            "EVENT_SINK_CLICKHOUSE_PII_MODELS",
+            ["user_profile", "external_id", "auth_user"],
         ),
-        ("EVENT_SINK_CLICKHOUSE_PII_MODELS", ["user_profile", "external_id"]),
```

Comment:
Ahh... I'm sorry I missed this change!

@Ian2012 Do you mind to change the names of the variables to make them a little less confusing?  Would prefer:

* `EVENT_SINK_MODELS_ENABLED`
* `EVENT_SINK_PII_MODELS`

Reply:
Sure